packages/business-rules/src/schemas/householder-appeal/generic-validators/date-validation.js

```diff
@@ -0,0 +1,11 @@
+const pinsYup = require('../../../lib/pins-yup');
+const parseDateString = require('../../../utils/parse-date-string');
+
+const dateValidation = (required) => {
+	if (required) {
+		return pinsYup.date().transform(parseDateString).required();
+	}
+	return pinsYup.date().transform(parseDateString).nullable();
+};
```

Comment:
i don't know how you feel about ternary operators, but this works too: 
`const dateValidation = (required = false) => {
	const validation = pinsYup.date().transform(parseDateString);

	return required ? validation.required() : validation.nullable();
};`

Reply:
(though I guess you don't need the required = false)